hack/tools/go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/joelanford/go-apidiff v0.4.0
 	github.com/onsi/ginkgo v1.16.5
 	gotest.tools/gotestsum v1.8.2
-	k8s.io/code-generator v0.24.4
-	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20220912130527-3c462340b297
+	k8s.io/code-generator v0.25.0
+	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20221008002318-c9b4a0f0fc38
```

Comment:
And hope this one pointing to `v1.2.x` branch

Reply:
Had updated to use latest version, verifying same.

Reply:
Fixed.